subsys/bluetooth/mesh/shell/shell.c

```diff
@@ -492,32 +492,46 @@ static void prov_node_added(uint16_t net_idx, uint8_t uuid[16], uint16_t addr,
 }
 
 #if defined(CONFIG_BT_MESH_PROVISIONER)
-static enum {
-	AUTH_NO_OOB,
-	AUTH_STATIC_OOB,
-	AUTH_OUTPUT_OOB,
-	AUTH_INPUT_OOB
-} auth_type;
+static const char * const output_meth_string[] = {
+	"Blink",
+	"Beep",
+	"Vibrate",
+	"Display_number",
+	"Display_string",
+};
+
+static const char *const input_meth_string[] = {
+	"Push",
+	"Twist",
+	"Enter_number",
+	"Enter_string",
+};
 
 static void capabilities(const struct bt_mesh_dev_capabilities *cap)
 {
-	if (cap->oob_type && auth_type == AUTH_STATIC_OOB) {
-		bt_mesh_auth_method_set_static(bt_mesh_shell_prov.static_val,
-					       bt_mesh_shell_prov.static_val_len);
-		return;
-	}
+	shell_print_ctx("Provisionee capabilities:");
```

Comment:
but what if a device doesn't support any OOB capabilities? Should the function go out before printing output\input capabilities? Otherwise, you will get printed the blank phrases.